corehq/apps/app_manager/views/utils.py

```diff
@@ -344,7 +349,7 @@ def update_linked_app(app, user_id):
                 _(
                     'This application uses mobile UCRs '
                     'which are not available in the linked domain: {ucr_id}'
-                ).format(str(e))
+                ).format(ucr_id=str(e))
```

Comment:
This is the typo I was referring to in the description, and the original error that led me to make this PR.